scripts/solve_network.py

```diff
@@ -159,6 +153,25 @@ def prepare_network(n, solve_opts):
 
 
 def add_CCL_constraints(n, config):

```

Comment:
In your experience, does the ci test these functions [not just this one]?
Indeed unit testing may by quite great to have or at least the validator similar to -eur

Reply:
CI definitely omits at least some of those functions. Agree that it would be a great application for CI. Happy to discuss the details